src/panels/lovelace/create-element/create-element-base.ts

```diff
@@ -50,14 +50,40 @@ const _createErrorElement = <T extends keyof CreateElementConfigTypes>(
   config: CreateElementConfigTypes[T]["config"]
 ): HuiErrorCard => createErrorCardElement(createErrorCardConfig(error, config));
 
+const _maybeCreate = <T extends keyof CreateElementConfigTypes>(
+  tag: string,
+  config: CreateElementConfigTypes[T]["config"]
+) => {
+  if (customElements.get(tag)) {
+    return _createElement(tag, config);
+  }
+
+  const element = _createErrorElement(
+    `Custom element doesn't exist: ${tag}.`,
+    config
```

Comment:
This should be the same error as here: https://github.com/home-assistant/home-assistant-polymer/pull/4857/files#diff-15d91f0ebd111fc153e36f517963b336R134

Reply:
Your link doesn't work anymore.